### PR TITLE
Sagonzal/lab infra

### PR DESCRIPTION
**Pull request by sangonzal**
This PR contains:
1) Integration with lab API
2) Integration with KeyVault
3) Integration tests for different scenarios

A couple of open items, which I'd like to get some input on and which we can discuss offline:
1) Where chromium.exe should go
2) IWA tests are currently failing. I believe we might not be able to automate this flow
3) Any scenarios that I might have missed and which we'd like to test

### CODE REVIEW

File: src/integrationtest/java/com.microsoft.aad.msal4j/AuthorizationCodeIT.java [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+        tcpListener = new TcpListener(queue);
+        tcpListener.startServer();
+    }
+
+    private String getResponseFromTcpListener(){
+        String response;
+        try {
+            response = queue.poll(20, TimeUnit.SECONDS);
+            if (response == null){
+                System.out.println("response is null");
```
[the remark this answers is not in the document]
Comment (SomkaPe) [reply src/integrationtest/java/com.microsoft.aad.msal4j/AuthorizationCodeIT.java]: :shipit:

**Comment by SomkaPe** [comment]
:shipit:

**Comment by henrik-me** [comment]
When you say OSS, do you use the governance tool for this?

---
In reply to: [468423260](https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/8#issuecomment-468423260) [](ancestors = 468423260)

File: pom.xml [hunk 2]
```diff
@@ -7,11 +7,9 @@
...
-    Azure active directory library for Java gives you the ability to add Windows Azure Active Directory 
-    authentication to your web application with just a few lines of additional code. Using our ADAL SDKs you 
-    can quickly and easily extend your existing application to all the employees that use Windows Azure 
-    AD and Active Directory on-premises using Active Directory Federation Services, including Office365 
-    customers.
+    Microsoft Authentication Library for Java gives you the ability to obtain tokens from Azure AD v2 (work and school
+		accounts, MSA) and Azure AD B2C, gaining access to Microsoft Cloud API and any other API secured by Microsoft
+		identities
   </description>
 	<url>https://github.com/AzureAD/azure-activedirectory-library-for-java</url>
```
Comment (henrik-me) [inline pom.xml]: >https://github.com/AzureAD/azure-activedirectory-library-for-java [](start = 6, length = 65)

should this also be MSAL?

**Comment by sangonzal** [comment]
> When you say OSS, do you use the governance tool for this?
> 
> In reply to: [468423260](https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/8#issuecomment-468423260) [](ancestors = 468423260)

I registered these in https://ossmsft.visualstudio.com/_oss for now. The new corporate governance will pick these up automatically when we merge into dev. If not, then I will manually add them there as well

File: pom.xml (hunk 2 above)
Comment (sangonzal) [reply pom.xml]: done

**Pull request merged**